.travis.yml

```diff
@@ -3,10 +3,18 @@ os: linux
 env:
 - DOCKER_COMPOSE_VERSION=1.17.1
 
+language: go
+go:
+- "1.10"
+
 before_install:
 - sudo rm /usr/local/bin/docker-compose
 - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
 - chmod +x docker-compose
 - sudo mv docker-compose /usr/local/bin
 
-script: make test
+install:
+- go get -t ./...
+- go get github.com/mattn/goveralls
+
+script: make test && make coveralls
```

Comment:
not sure is it better to exec as docker-compose command, or as host command after all. both sounds ok?